src/website/assets/scripts/chi-docs.js

```diff
@@ -108,6 +108,13 @@ onLoad(() => {
     };
   });
 
+  Array.prototype.forEach.call(
+    document.getElementsByClassName("m-dropdown__trigger"),
+    function(el){
+      chi.dropdown(el);
```

Comment:
This code is going to make the dropdowns in the /component/dropdowns/ page work. Is this the desired behavior?
The dropdown in the /javascript/dropdown/ page is going to be initialized twice. It is safe as the Dropdown code will detect it, but it is not a best practice. Maybe we can globally instantiate only the version dropdown and let other pages initiate their own dropdowns: 

`chi.dropdown(document.getElementById('version-dropdown'));`

The other solution would be to keep this code as it is and remove any other code that instantiates dropdowns.

Reply:
Apart from the revision, changing the subject, since [[CHI-137] Add floating label support to inputs](https://github.com/CenturyLinkCloud/ux-chi/commit/ad1b9245ea2e8c919553686e228d44e0abfb34f7), factory methods for the JavaScript components accept an array of DOMElements, so might use:
```
chi.dropdown(document.getElementsByClassName("m-dropdown__trigger"));
```
instead of the forEach loop.

Reply:
@dani-cl-madrid 👍